### ISSUE TITLE
Tf implementation of ShuffleV2Block

### ISSUE THREAD

**Issue by AlexisPapaioannou** [deepcam-cn/yolov5-face#74]
Is there any implementation of the ShuffleV2Block in TensorFlow, in order to convert the PyTorch model to tf.
